workspaces/btpsa.code-workspace

```diff
@@ -0,0 +1,19 @@
+{
+    "folders": [
+        {
+            "path": "../",
+            "name": "SAP Cloud SDK Quickstart - SAP BTP"
+        }
+    ],
+    "settings": {
+        "files.exclude": {
```

Comment:
Also exclude the devcontainer files of the other flavours?

Reply:
Good point: this is introduced with commit <https://github.com/Azure-Samples/app-service-javascript-sap-cloud-sdk-quickstart/pull/41/commits/d4d81a02485c74ffe7135279f2a20b25683ce4fa>